### PR TITLE
REL: Welcome to v0.13.0

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

Please check the type of change your PR introduces:

- [x] Other (please describe): Release!

## Pull request checklist

This is a release PR, all changes were already approved previously ;)
Tests are passing, of course, and docs were updated.

## What is the current behavior?
The current master branch is outdated. A lot of changes have been made to develop branch and, finally, I believe that we are able to merge everything to master, upgrading our python package version.

## What is the new behavior?
Several changes, really! I will try to summarize them below:
* Environment Analysis 2.0: improved, tested, and examplified.
* Fins sweep angle implemented
* Integration with Windy/ECMWF
* New AeroSurfaces class
* postProcess method is died, say hi to cached properties!
* Timezonefinder is now optional

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Please wait for merge conflicts to be solved before reviewing!

### CODE REVIEW

**Comment by Gui-FernandesBR** [comment]
Conflicts were solved and tests are running,

@giovaniceotto and @MateusStano could you guys review this one pls? 
After merging, we can create a new release so the PyPI package can be updated.

I'll wait for review of both of you before merging, this is a large PR.

File: rocketpy/Flight.py
Comment (MateusStano) [inline rocketpy/Flight.py:2277]:
```suggestion
    def retrieve_acceleration_arrays(self):
```

Does this need to be a `@cached_property`?

**Review by MateusStano — CHANGES_REQUESTED** [review]

File: data/dispersion csv files/Valetudo.csv
Comment (giovaniceotto) [inline data/dispersion csv files/Valetudo.csv:1]: Is this file crucial for this release? If this is not the case, consider deleting it to prevent accumulating unnecessary files.

File: docs/user/requirements.rst
Comment (giovaniceotto) [inline docs/user/requirements.rst:51]:
```suggestion
```

File: data/dispersion csv files/Valetudo.csv
Comment (giovaniceotto) [reply data/dispersion csv files/Valetudo.csv:1]: Furthermore, the standard for CSV files is to be comma delimited. So every ";" should be replaced by ",".

File: tests/fixtures/dispersion/Valetudo_inputs.csv
Comment (giovaniceotto) [inline tests/fixtures/dispersion/Valetudo_inputs.csv:1]: Consider using "," instead of ";", as it is the standard for CSV files.

File: rocketpy/utilities.py
Comment (giovaniceotto) [inline rocketpy/utilities.py:205]: Why is this the case?

File: rocketpy/utilities.py
Comment (giovaniceotto) [inline rocketpy/utilities.py:223]: Which resources do not work?

File: rocketpy/utilities.py
Comment (giovaniceotto) [inline rocketpy/utilities.py:215]: Please describe the "format" of this dictionary. Consider adding examples of when this utility can be useful in the docstring.

**Review by giovaniceotto — CHANGES_REQUESTED** [review]

**Comment by giovaniceotto** [comment]
@Gui-FernandesBR, here is my first review. As this is quite a large PR, I decided to review half of the files today and will review the other half tomorrow, although I have already looked through all the files. Let me know if you have any questions about my comments.

File: rocketpy/Flight.py
Comment (Gui-FernandesBR) [reply rocketpy/Flight.py:2277]: The retrieve_acceleration_arrays property is used 6 times at the Flight.py file.
If we don't use it as a cached_property, what you happen is that the functions inside the `def retrieve_acceleration_arrays()` will be called six times, which can significantly impact performance.

I can imagine that better names to this function can be given, if you present some suggestions we can discuss.

File: rocketpy/Flight.py
Comment (Gui-FernandesBR) [reply rocketpy/Flight.py:2329]: Same as above

File: rocketpy/Flight.py
Comment (Gui-FernandesBR) [reply rocketpy/Flight.py:2422]: Same as above

File: data/dispersion csv files/Valetudo.csv
Comment (Gui-FernandesBR) [reply data/dispersion csv files/Valetudo.csv:1]: We need this file to run the `test_create_dispersion_dictionary` function. It is not a duplicated file, neither a large one, so I would say it is not a big deal having it here.

Regarding the semicolon problem, well..., it was a workaround since there're some items that present commas, as you can see at the ensembleMember, which contains a list.
I think the best way would be having these information as .json files. However, it is not so human-readable as the csv one, so I don't know. I rather take few hours to think what is the best solution here.

File: rocketpy/Flight.py
Comment (MateusStano) [reply rocketpy/Flight.py:2277]: Oh okay. This makes sense then, no problems here

File: data/dispersion csv files/Valetudo.csv
Comment (giovaniceotto) [reply data/dispersion csv files/Valetudo.csv:1]: YAML files would be the best option in my opinion. I'll raise an issue regarding this, but the implementation works, so I do not believe we should hold back this PR.

However, I do believe the docstring should give a better explanation of why you should use ";".

**Comment by giovaniceotto** [comment]
I'll need one more day to run a couple of checks before we are ready to go. Almost there!

File: data/dispersion csv files/Valetudo.csv
Comment (Gui-FernandesBR) [reply data/dispersion csv files/Valetudo.csv:1]: The idea of using csv files was to have an easy interface, allowing the user to write different csv files from excel, for example, and running a batch of simulations with them. YAML does not allow that.

Anyway, I agree we should not spend too many time with this method. It was created mainly to solve a specific problem last year, which was already addressed,

**Comment by giovaniceotto** [comment]
Here is a problem with the documentation, which can be checked here: https://docs.rocketpy.org/en/develop/user/index.html

It seems the Environment Analysis notebook is using H1 headers instead of H2 for Surface Level Analysis, Pressure Level Analysis and Going Further with your Analysis, which causes the main index of the user guide page to get cluttered.

Furthermore, the utilities model notebook is out of date regarding the new function `create_dispersion_dictionary`.

When these issues (and the docstring for `create_dispersion_dictionary` function) are fixed, I will accept the changes and this PR will be able to be merged.

**Comment by Gui-FernandesBR** [comment]
Ok, now tests are passing, and the documentation seems to be good. I also added two notebooks to index.rst file, what I cannot is saying why they weren't included before.

> Furthermore, the utilities model notebook is out of date regarding the new function create_dispersion_dictionary

Hey, I don't think it should be mandatory documenting all the new utilities functions on notebooks. Other classes don't even have a notebook.... Therefore, I vote for skipping this one. Maybe in the future we can provide that. The current utilities module notebook seems good to me.

Everything else was addressed, at least temporarily.

Btw sorry for long commit list, it turns out some tests were passing locally without running so I committed some poor lines thinking that it was running well.

**Comment by giovaniceotto** [comment]
@Gui-FernandesBR, I have fixed the documentation problem related to headings 1 being used mistakenly. I have also defined which exception to catch in the `create_dispersion_dictionary` function and replaced prints with warnings. Please review this changes.

Once done, I believe we are ready to merge. Quick idea: wait until the weekend (just one day) so that we can host a launch party in our discord channel with the new members so that they can see how a release is done (and so that we can promote this new version together)!

**Review by giovaniceotto — APPROVED** [review]

**Comment by Gui-FernandesBR** [comment]
I got an idea: use quotes to separate objects in the csv file!!

**Comment by MateusStano** [comment]
We are releasing this today!

**Pull request merged**
